# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1256148813 25200
#      Wed Oct 21 11:13:33 2009 -0700
# Node ID 7bf4f3d6495547ee27d37eb1b9fb24128028725b
# Parent  807a3720e6e24d411c9220ec7520abc209a806c7
Fix unbalanced parentheses warning during creation of pdf documentation.

deftypefnx for who and whos needed a space between function name and
parameter list in order for TeX macro to work correctly

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2009-10-21  Rik <octave@nomad.inbox5.com>
+
+	* variables.cc: Correct use of deftypefnx for who and whos
+
 2009-10-21  John W. Eaton  <jwe@octave.org>
 
 	* load-path.h (load_path::dir_path::dir_time_last_checked): New field.
 	(load_path::dir_path::dir_path, load_path::dir_path::operator=):
 	Initialize and copy dir_time_last_checked.
 	* load-path.cc 	(load_path::dir_info::initialize): Store time of
 	last update.
 	(load_path::dir_info::update): Check directory modification time
diff --git a/src/variables.cc b/src/variables.cc
--- a/src/variables.cc
+++ b/src/variables.cc
@@ -1601,17 +1601,17 @@ do_who (int argc, const string_vector& a
   return retval;
 }
 
 DEFUN (who, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn  {Command} {} who\n\
 @deftypefnx {Command} {} who pattern @dots{}\n\
 @deftypefnx {Command} {} who option pattern @dots{}\n\
-@deftypefnx {Command} {C =} who(\"pattern\", @dots{})\n\
+@deftypefnx {Command} {C =} who (\"pattern\", @dots{})\n\
 List currently defined variables matching the given patterns.  Valid\n\
 pattern syntax is the same as described for the @code{clear} command.\n\
 If no patterns are supplied, all variables are listed.\n\
 By default, only variables visible in the local scope are displayed.\n\
 \n\
 The following are valid options but may not be combined.\n\
 \n\
 @table @code\n\
@@ -1649,17 +1649,17 @@ matching the given patterns.\n\
   return retval;
 }
 
 DEFUN (whos, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn  {Command} {} whos\n\
 @deftypefnx {Command} {} whos pattern @dots{}\n\
 @deftypefnx {Command} {} whos option pattern @dots{}\n\
-@deftypefnx {Command} {S =} whos(\"pattern\", @dots{})\n\
+@deftypefnx {Command} {S =} whos (\"pattern\", @dots{})\n\
 Provide detailed information on currently defined variables matching the\n\
 given patterns.  Options and pattern syntax are the same as for the\n\
 @code{who} command.  Extended information about each variable is\n\
 summarized in a table with the following default entries.\n\
 \n\
 @table @asis\n\
 @item Attr\n\
 Attributes of the listed variable.  Possible attributes are:\n\
